src/main/clojure/clara/rules/durability/fressian.clj

```diff
@@ -535,17 +539,34 @@
           indexed-facts))))
 
   (deserialize [_ mem-facts opts]
+    ;; Options can be provided via opts.
```

Comment:
This seems like it should be documented as doc strings at the protocol function `deserialize` level in `clara.rules.durability`.